projects/ontimize-web-ngx/src/lib/components/table/o-table.component.ts

```diff
@@ -2952,12 +2953,28 @@ export class OTableComponent extends AbstractOServiceComponent<OTableComponentSt
    * @returns column data by attr
    */
   getColumnDataByAttr(attr, row: any): any {
+    let operation: string = "";
+    if (attr.split('-')) {
```

Comment:
It is necessary to find another way of identifying column and its group date operation. If the attr of the column contains hyphens this algorithm will fail.